src/functions/src/submitContactForm.ts

```diff
@@ -1,10 +1,46 @@
 import axios from 'axios';
 import * as admin from 'firebase-admin';
 import * as functions from 'firebase-functions';
+import { ApolloClient, createHttpLink, gql, InMemoryCache } from '@apollo/client';
+import { setContext } from '@apollo/client/link/context';
+// import { ApolloClient, createHttpLink, InMemoryCache, gql } from "@apollo/client";
+// import { setContext } from '@apollo/client/link/context';
 
 if (admin.apps.length === 0) {
   admin.initializeApp();
 }
+export async function createDiscussion(topic: string, body: string) {
+  const httpLink = createHttpLink({
+    uri: 'https://api.github.com/graphql',
```

Comment:
when running i get this error: 
> Unhandled error Invariant Violation: \n\"fetch\" has not been found globally and no fetcher has been configured. To fix this, install a fetch package (like https://www.npmjs.com/package/cross-fetch), instantiate the fetcher, and pass it into your HttpLink constructor. For example:\n\nimport fetch from 'cross-fetch';\nimport { ApolloClient, HttpLink } from '@apollo/client';\nconst client = new ApolloClient({\n  link: new HttpLink({ uri: '/graphql', fetch })\n});

so i think you need to add a `fetch` here... (btw what's the difference between `createHttpLink` and `new HttpLink`? i'm not familiar with graphql/apollo client)